jenkinsfiles/cordova_jenkinsfile.groovy

```diff
@@ -1,14 +1,72 @@
 /**
 * Cordova Jenkinsfile
 */
-node('android') {
-   stage 'Checkout'
-	 checkout scm
 
-	 stage 'Prepare'
-	 cordova platform add android
-	 cordova prepare
+def platform = params?.PLATFORM?.trim()
+def buildType = params?.BUILD_CONFIG?.trim()
+def keystore_path = "/opt/android-sdk-linux/android.debug"
 
-	 stage 'Build'
-	 cordova build android
+if(platform == 'android') {
```

Comment:
can we move this platform check into the stage blocks when its required (prepare and archive)?